refact_webgui/webgui/static/tab-finetune.js

```diff
@@ -387,73 +300,22 @@ function render_checkpoints(data = []) {
             if(cell.dataset.checkpoint === finetune_configs_and_runs.active.specific_checkpoint) {
                 row.classList.add('table-success');
             }
-            const activate_cell = document.createElement('td');
             const download_cell = document.createElement('td');
 
-            activate_cell.innerHTML = `
-                <button class="btn btn-hover btn-primary btn-sm">
-                <i class="bi bi-play-fill"></i>
-                </button>`;
             download_cell.innerHTML = `
                 <a href="/lora-download?run_id=${selected_lora}&checkpoint_id=${element.checkpoint_name}"
                    download class="btn btn-hover btn-primary btn-sm">
                 <i class="bi bi-download"></i>
                 </a>`;
 
-            row.appendChild(activate_cell);
             row.appendChild(download_cell);
             row.appendChild(cell);
 
             checkpoints.appendChild(row);
-            activate_cell.addEventListener('click', (event) => {
-                if(!row.classList.contains('table-success')) {
-                    let prev = document.querySelector('.table-checkpoints .table-success');
-                    if (prev) {
-                        prev.classList.remove('table-success');
-                    }
-                    row.classList.add('table-success');
-                }
-                finetune_activate_run(selected_lora, cell.dataset.checkpoint);
-            });
-            activate_cell.addEventListener('click', (event) => {
-                event.stopPropagation();
-            });
         });
     }
 }
 
-function animate_use_model() {
-    use_model_panel.classList.add('animate-pane');
-    setTimeout(() => {
-        use_model_panel.classList.remove('animate-pane');
-    }, 1000);
-}
-
-function finetune_switch_activate(lora_mode, run_id, checkpoint) {
-    animate_use_model();
-    const finetune_model = finetune_configs_and_runs.completion_model.finetune;
-    let send_this = {
-        "model": finetune_model,
-        "lora_mode": lora_mode,
-        "specific_lora_run_id": run_id ? run_id : finetune_configs_and_runs.active[finetune_model].specific_lora_run_id,
```

Comment:
this code still in finetune tab js
if(cell.dataset.checkpoint === finetune_configs_and_runs.active.specific_checkpoint) {